### PR TITLE
something about PRs

### PR DESCRIPTION
**Pull request by nichlas0010**

### CODE REVIEW

File: models/Github/GithubManager.js [hunk 1]
```diff
@@ -140,7 +140,10 @@ class GithubManager {
     }
 
     for (var channel of discordSubsystem.getPrimaryGuild().channels.array()) {
-      if (channel.id == config.discord_github_channel) {
+      if (channel.id == config.discord_github_channel && payload.sender.login == "yogstation13-bot") {
```
Comment (AshCorr) [inline models/Github/GithubManager.js]: uhhm. what?

Comment (nichlas0010) [reply models/Github/GithubManager.js]: Making it so only yogbot PRs get posted in #botspam while all other PRs (IE PRs made by hoomans, not mirrorbots are posted in #coder-public

Comment (AshCorr) [reply models/Github/GithubManager.js]: You could also check if the PR's title starts with [MIRROR]

Comment (nichlas0010) [reply models/Github/GithubManager.js]: We don't have any other bots though...

**Pull request merged**